modules/apps/29-fee/keeper/escrow_test.go

```diff
@@ -113,8 +155,8 @@ func (suite *KeeperTestSuite) TestDistributeFee() {
 				forwardRelayer = "invalid address"
 			},
 			func() {
-				// check if the refund acc has been refunded the timeoutFee & recvFee
-				expectedRefundAccBal := refundAccBal.Add(defaultTimeoutFee[0]).Add(defaultRecvFee[0]).Add(defaultTimeoutFee[0]).Add(defaultRecvFee[0])
+				// check if the refund acc has been refunded the recvFee
+				expectedRefundAccBal := refundAccBal.Add(defaultRecvFee[0]).Add(defaultRecvFee[0])
```

Comment:
It should be `recv_fee` twice, this is because, in this test case, only the forward relayer address is invalid, therefore only the `recv_fee` is refunded. The reason it is added twice is that, packetFee is paid twice in line 153